package.json

```diff
@@ -78,6 +78,7 @@
     "rimraf": "^2.3.2",
     "serve-favicon": "^2.1.5",
     "strong-control-channel": "^2.0.0",
+    "strong-deploy-receivers'": "^1.0.0",
```

Comment:
You've got a trailing `'`.. that's what is breaking CI, it's treating it as a path because `'` isn't a valid package name character.